### PR TITLE
Add language client sample

### PR DESCRIPTION
**Pull request by jbolinger**
A simple language example using the google-cloud-java client library.

/cc @neozwu 

Do not merge until all three samples have been reviewed and the top level `README.md` is updated.

### CODE REVIEW

File: nl/LanguageClient/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+This directory contains an Android example that uses the
+[Cloud Natural Language API](https://cloud.google.com/natural-language/)
+with the [Google Cloud Client Library for Java]
+
+## Prerequisites
+
+### Enable the Data Loss Prevention API
+
+If you have not already done so,
+[enable the Natural Language API for your project](https://cloud.google.com/natural-language/docs/quickstart-client-libraries).
```
Comment (dzlier-gcp) [inline nl/LanguageClient/README.md]: As with DLP sample, do we want this link to go to the CLI quickstart, or directly to the enable-api page for the user? The language seems to indicate the latter. Except in this case, the quickstart page doesn't even seem to have its own link to enable the API.
https://pantheon.corp.google.com/flows/enableapi?apiid=language.googleapis.com

Comment (jbolinger) [reply nl/LanguageClient/README.md]: Will do (although I'll use the external link: https://console.cloud.google.com/flows/enableapi?apiid=language.googleapis.com)

File: nl/LanguageClient/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+resources as `app/src/main/res/raw/credential.json`.
+
+Again, ***you should not do this in your production app.***
+
+See the [Cloud Platform Auth Guide](https://cloud.google.com/docs/authentication#developer_workflow)
+for more information.
+
+## Client Libraries on Android
+
+The [Google Cloud Client Library for Java] that is used for this example can be used on Android,
```
Comment (davidcavazos) [inline nl/LanguageClient/README.md]: Missing link?

Comment (jbolinger) [reply nl/LanguageClient/README.md]: yup, fixed.

**Pull request merged**